src/openforms/js/components/admin/form_design/variables/VariablesTable.js

```diff
@@ -138,7 +168,7 @@ const EditableVariableRow = ({index, variable, onDelete, onChange}) => {
           <Select
             name="formDefinition"
             choices={formStepsChoices}
-            value={variable.formDefinition}
+            value={variable.formDefinition || ''}
```

Comment:
perhaps we should fix that in the `Select` component itself - if the prop `value` is `null`, replace it with empty string?